src/components/stage/stage.jsx

```diff
@@ -66,13 +68,22 @@ const StageComponent = props => {
                         <Loupe colorInfo={colorInfo} />
                     </Box>
                 ) : null}
-                {question === null ? null : (
-                    <div
-                        className={classNames(
-                            styles.stageOverlayContent,
-                            styles.stageOverlayContentBorderOverride
-                        )}
-                    >
+                <div
+                    className={styles.stageBottomWrapper}
+                    style={{
```

Comment:
re: RTL handling, I made it switch sides at first, but then after talking it over with ChrisG, we decided to leave it as is- it is a visual feature that shows up in a fixed position on the stage, so people might make artwork around it or pointing to it, which would be broken if it flipped in RTL (in the commit history you can see "Position mic indicator in RTL" and then "Actually, don’t move indicator in RTL).

re: in a CSS file, my understanding is that because we are using the passed in stageDimensions, they have to be out here. The "left: 50%" part is so tightly coupled to the "marginLeft" part (it's a clever trick basically) that it seems they should be in the same place if we want it to be at all understandable.

Reply:
👍